ffwd-http-reporter/src/main/java/com/spotify/metrics/ffwdhttp/FastForwardHttpReporter.java

```diff
@@ -440,6 +438,22 @@ public void stop() {
         }
     }
 
+    public void stopWithFlush() {
+        if (scheduledFuture != null) {
+            scheduledFuture.cancel(false);
+        }
+        if (executorOwner) {
+            executorService.shutdown();
+        }
+        //final flush of metrics
```

Comment:
I wonder if the log line might be sufficient documentation for what's going on.

Reply:
true haha